Gems/GraphCanvas/Code/Source/Components/Slots/Execution/ExecutionSlotLayoutComponent.cpp

```diff
@@ -28,6 +30,14 @@ namespace GraphCanvas
 
         m_slotConnectionPin = aznew ExecutionSlotConnectionPin(owner.GetEntityId());
         m_slotText = aznew GraphCanvasLabel();
+
+        if (const AZ::Entity* ownerEntity = owner.GetEntity())
+        {
+            if (const SlotComponent* slotComponent = ownerEntity->FindComponent<ExecutionSlotComponent>())
+            {
+                m_isNameHidden = slotComponent->IsNameHidden();
+            }
+        }
```

Comment:
Similar as above, can this be passed down to `ExecutionSlotLayout` instead of having it query up from its owner?